tools/Importer/Importer.cs

```diff
@@ -408,11 +409,21 @@ private static IEnumerable<string> GetLinesInBlobRange(IList<BlobItem> blobs, st
             Console.WriteLine($"{logPrefix}: Completed reads. Total={lines} secs={(int)sw.Elapsed.TotalSeconds} speed={(int)(lines / sw.Elapsed.TotalSeconds)} lines/sec");
         }
 
-        private static BlobContainerClient GetContainer(string connectionString, string containerName)
+        private static BlobContainerClient GetContainer(string connectionString, string containerName, string storageAccountName)
         {
             try
             {
-                var blobServiceClient = new BlobServiceClient(connectionString);
+                BlobServiceClient blobServiceClient;
+                if (!string.IsNullOrEmpty(connectionString))
+                {
+                    blobServiceClient = new BlobServiceClient(connectionString);
+                }
+                else
+                {
+                    var defaultAzureCredential = new DefaultAzureCredential();
```

Comment:
NIT - would be nice to supply options to the `DefaultAzureCredential` like elsewhere in this file with `FhirAuthCredentialOptions`.